src/connecpy/_server_sync.py

```diff
@@ -106,22 +116,13 @@ def prepare_response_headers(
     return headers, use_compression
 
 
-def read_chunked(input_stream):
-    chunks = []
+def _read_body(environ: WSGIEnvironment) -> Iterator[bytes]:
+    input_stream: BytesIO = environ["wsgi.input"]
     while True:
-        line = input_stream.readline()
-        if not line:
-            break
-
-        chunk_size = int(line.strip(), 16)
```

Comment:
I couldn't find any documentation of this sort of behavior, it seems even if the WSGI server handles chunked requests, it would present the final bytestream to the app.